src/openms/include/OpenMS/SYSTEM/BuildInfo.h

```diff
@@ -160,6 +160,14 @@ namespace OpenMS
         return 1;
         #endif
       }
+      /// @brief Set the number of threads that OpenMP will use (including hyperthreads)
+      /// Note: This could also be limited by the OMP_NUM_THREADS environment variable
```

Comment:
what does "could" mean here? Is it documented what takes precedence?

Reply:
yes, it is documented by openmp

Reply:
the env variable of course initializes the variable. whatever you set in code will overwrite it.

Reply:
because "limit" sounds like it sets an upper limit